servant-server/src/Servant/Server/Internal.hs

```diff
@@ -533,6 +534,18 @@ instance HasServer api context => HasServer (HttpVersion :> api) context where
   route Proxy context subserver =
     route (Proxy :: Proxy api) context (passToServer subserver httpVersion)
 
+-- | Ignore @'Summary'@ in server handlers.
```

Comment:
Yeah I understand. We'd need some kind of `mapError` post phase in `Delayed` to do that. Can you or @fierce-katie open a ticket about that. Let's not complicate this PR